site/assets/scss/_content.scss

```diff
@@ -2,13 +2,12 @@
 // Bootstrap docs content theming
 //
 
-.bd-content {
-  // Offset content from fixed navbar when jumping to headings
-  > :target {
-    padding-top: 5rem;
-    margin-top: -5rem;
-  }
+// Offset content from fixed navbar when jumping to headings
+:root {
+  scroll-padding-top: 5rem;
+}
 
+.bd-content {
   > h2:not(:first-child) {
```

Comment:
Maybe just `> h2` could be a quick fix to handle it on a stable version (even if it changes a bit the first `h2` `margin-top`)